e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/aiclassification/arithmetic/ACArithmeticEMTest.kt

```diff
@@ -71,21 +85,22 @@ class ACArithmeticEMTest : AIClassificationEMTestBase() {
 
             val ok = listOf(
                 ptr.fromVerbPath("GET","/api/arithmetic",
-                    queryParams = mapOf("x" to "42", "y" to "-4")
+                    queryParams = mapOf("x" to "-42", "y" to "4")
```

Comment:
these changes should be in a different PR, as i need to double-check them by running code